src/pages/components/text-inputs.mdx

```diff
@@ -0,0 +1,199 @@
+property.
+
+Regular labels are not recommended for text inputs, as they flatten visual hierarchy making
+scanning more difficult.
+
+import RegularWeightLabel from '../../examples/text-inputs/RegularWeightLabel.tsx';
+import RegularWeightLabelCode from '!!raw-loader!../../examples/text-inputs/RegularWeightLabel.tsx';
+
+<CodeExample code={RegularWeightLabelCode}>
+  <RegularWeightLabel />
+</CodeExample>
+
+### Hint
+
+Hint adds additional context to the text Input’s label.
+
+import Hint from '../../examples/text-inputs/Hint.tsx';
+import HintCode from '!!raw-loader!../../examples/text-inputs/Hint.tsx';
+
+<CodeExample code={HintCode}>
+  <Hint />
+</CodeExample>
+
+### Size
+
+A text Input can be default or compact.
+
+import Size from '../../examples/text-inputs/Size.tsx';
+import SizeCode from '!!raw-loader!../../examples/text-inputs/Size.tsx';
+
+<CodeExample code={SizeCode}>
+  <Size />
+</CodeExample>
+
+### Bare
+
+Input borders and padding can be removed using the `isBare` property. Use an
+[aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute)
+to label the input.
+
+import Bare from '../../examples/text-inputs/Bare.tsx';
+import BareCode from '!!raw-loader!../../examples/text-inputs/Bare.tsx';
+
+<CodeExample code={BareCode}>
+  <Bare />
+</CodeExample>
+
+### Focus inset
+
+Box-shadow focus styling can be inset using the `focusInset` property.
+
+This keeps the focus contained within the input area when it is flush with another component.
+
+import FocusInset from '../../examples/text-inputs/FocusInset.tsx';
+import FocusInsetCode from '!!raw-loader!../../examples/text-inputs/FocusInset.tsx';
+
+<CodeExample code={FocusInsetCode}>
+  <FocusInset />
+</CodeExample>
+
+### Placeholder
+
+Placeholder text gives an example of content format. See [Placeholder text](#TODO) standard for
+best practices.
+
+import Placeholder from '../../examples/text-inputs/Placeholder.tsx';
+import PlaceholderCode from '!!raw-loader!../../examples/text-inputs/Placeholder.tsx';
+
+<CodeExample code={PlaceholderCode}>
+  <Placeholder />
+</CodeExample>
+
+### Validation
+
+Success, warning, or danger validation messages are shown using the Message component with the text
+Input.
+
+import Validation from '../../examples/text-inputs/Validation.tsx';
+import ValidationCode from '!!raw-loader!../../examples/text-inputs/Validation.tsx';
+
+<CodeExample code={ValidationCode}>
+  <Validation />
+</CodeExample>
+
+### Inline
+
+An Input uses `display:block` by default. It can use `display:inline`. See example code for inline
+styling overrides.
+
+import Inline from '../../examples/text-inputs/Inline.tsx';
+import InlineCode from '!!raw-loader!../../examples/text-inputs/Inline.tsx';
+
+<CodeExample code={InlineCode}>
+  <Inline />
+</CodeExample>
+
+### Disabled
+
+A disabled Input is not interactive, and not perceived by a screen reader. See [Disabled UI](#TODO)
```

Comment:
What is the v1 plan for missing pattern guidelines?

Reply:
I think the guidelines could be flushed out after v1 (if there are guidelines). This copy has been revised.